### PR TITLE
Remove pi as a default answer form in numeric-input

### PR DESCRIPTION
**Pull request by handeyeco**
## Summary:
This is kind of complicated and a super edge-case.

- IF a content creator didn't set an explicit answer form for a question using numeric-input
- AND the correct answer was close to a multiple of `pi/12`
- AND the submitted answer was within a small margin of error to the correct answer
- THEN the validator would think the learner was approximating pi - even if the question had nothing to do with pi

There's a lot of conversation in the ticket, but the tl;dr is we decided not to validate against pi unless it was explicitly set as an option.

Issue: LC-691

## Test plan:

Original steps to repro:
1. Access Khan Academy.
2. Go to the exercise "Subtracting decimals: thousandths"
3. Look for this specific question: 52.389 – 7.1
4. Type anything between 45.282 and 45.288 as its answer.
5. This notification will come from the check button: `Your answer is close, but may have approximated pi. Enter your answer as a multiple of pi, like <code>12\ \text{pi}</code> or <code>2/3\ \text{pi}</code>.`

Thankfully for the learners and unfortunately for those who want to manual test this, Charlie explicitly set the answerForm for this question so we can't reproduce with those steps anymore. It's so edge-case that I'm not sure how we'd intentionally find a live example of the problem, so I wrote a unit test.

### CODE REVIEW

File: packages/perseus/src/widgets/__tests__/numeric-input.test.ts [hunk 1]
```diff
@@ -212,6 +213,40 @@ describe("static function validate", () => {
...
+                    value: 45.289,
+                },
+            ],
+            labelText: "",
+            size: "normal",
+            static: false,
+            coefficient: false,
+        };
+
+        const useInput = {
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/__tests__/numeric-input.test.ts:240]: ? 

```suggestion
        const userInput = {
```

File: packages/perseus/src/widgets/__tests__/numeric-input.test.ts [hunk 2]
```diff
@@ -212,6 +213,40 @@ describe("static function validate", () => {
...
+            currentValue: "45.282",
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/__tests__/numeric-input.test.ts:241]: I noticed in the PR description that you talk about the answer being close to some factor if `pi/12`.  

Is this, or the value on line 231 some factor of pi?  If so, could you note that in comments here (saves time instead of having to look up the bug in Jira)?

**Review by jeremywiebe — APPROVED** [review]
